docs/types/value-types.rst

```diff
@@ -456,6 +456,15 @@ String literals are written with either double or single-quotes (``"foo"`` or ``
 
 For example, with ``bytes32 samevar = "stringliteral"`` the string literal is interpreted in its raw byte form when assigned to a ``bytes32`` type.
 
+.. note::
+
+  While Solidity does not have string manipulation functions, you can use
```

Comment:
Actually I think this note should rather go to string types and not to string literal types.

Reply:
https://solidity.readthedocs.io/en/develop/types.html#arrays - we could introduce a small header there.

Reply:
OK done, I'm going to revisit this for some polish later, as I feel strings are a bit confusing for some…